src/search/_base.scss

```diff
@@ -49,4 +49,72 @@
 			margin-right: auto;
 		}
 	}
+
+	//new styles from the update
+	h2 {
+		font-family: "Noto sans", sans-serif;
+		font-size: 20px;
+		line-height: 34px;
+		padding-bottom: 8px;
+		padding-top: 8px;
+	}
+
+	h3 {
+		font-family: "Noto sans", sans-serif;
+		font-size: 24px;
+	}
+
+	.label li {
+		background-color: #5e738b;
+		color: white;
+		font-size: 16px;
+		font-weight: bold;
+		margin-top: 5px;
+		padding-left: 5px;
+		padding-right: 5px;
+		//this should also include CSS to cut off and add an expand/collapse, when there are more labels than fit on 1 row
+	}
+
+	.expand-collapse-buttons {
+		color: white;
+		font-size: 16px;
+		font-weight: bold;
+		padding-left: 5px;
+		padding-right: 5px;
+	}
+
+	.breadcrumb li {
+		color: #006621;
+		display: inline-block;
+		font-size: 20px;
+		margin-top: 10px;
+	}
+
+	.breadcrumb li:first-child::after {
+		content: ">";
+	}
+
+	cite a {
+		color: #006621;
+		display: inline-block;
+		font-size: 20px;
+		margin-top: 10px;
+	}
+
+	p {
+		font-family: "Noto sans", sans-serif;
+		font-size: 20px;
+		line-height: 34px;
+		padding-bottom: 8px;
+		padding-top: 8px;
+	}
+
+	time {
```

Comment:
Styling the section element like that might be not optimal and could cause some undesirable effect. It would worth it to target that style with a CSS class or to ensure it is in-scope.

Reply:
added a search-description class